Test/Plaxis1DVerif-qstatic.reg

```diff
@@ -0,0 +1,26 @@
+Plaxis1DVerif.xinp -2D -mixed-full
+
+	Topology sets: Bottom (1,3,1D)
+	               Top (1,4,1D)
+	               Vertical (1,1,1D) (1,2,1D)
+Elasticity: 2D, gravity = 0 9.81
+	Density of Fluid, rhof = 1000
+	Density of Solid, rhos = 2700
+	Bulk Modulus of Fluid, Kf = 1e+99
+	Bulk Modulus of Solid, Ks = 1e+99
+	Bulk Modulus of Medium, Ko = 0
+	Young's Modulus, E = 1e+06
+	Poisson's Ratio, nu = 0
+	Porosity, n = 0.5
+Number of elements    81
+Number of nodes       265
+Number of dofs        409
+Number of D-dofs      288
+Number of P-dofs      121
+Number of unknowns    352
+  step=1  time=8475.82
+  Primary solution summary: L2-norm            : 0.430464
+                            Max Y-displacement : 1 node 144
```

Comment:
Is there a reason why the pressure isn't printed if it's zero? Seems like a sensible thing to check here.

Reply:
No particular reason, guess it was that someone thought it was unnecessary to check quantities that should be identically zero (inherited from Elasticity). But I agree that for the pressure it is wise to check this. So replace SIMPoroElasticity.h:l88-90 with the following:
```
     IFEM::cout <<"\n                            Max pressure       : "
                << utl::trunc(dMax[nsd]) <<" node "<< iMax[nsd] <<"\n";'
```
We still need the utl::trunc deviise to make sure values less than a certain tolerance is printed as 0.